pandas/tests/reshape/test_reshape.py

```diff
@@ -218,35 +218,51 @@ def test_multiindex(self):
 
 class TestGetDummies(object):
 
-    sparse = False
-
-    def setup_method(self, method):
-        self.df = DataFrame({'A': ['a', 'b', 'a'],
-                             'B': ['b', 'b', 'c'],
-                             'C': [1, 2, 3]})
-
-    def test_basic(self):
+    @pytest.fixture
+    def df(self):
+        return DataFrame({'A': ['a', 'b', 'a'],
+                          'B': ['b', 'b', 'c'],
+                          'C': [1, 2, 3]})
+
+    @pytest.fixture(params=['uint8', 'int64', np.float64, bool, None])
+    def dtype(self, request):
```

Comment:
If we use this fixture in many places it's going to add a ton of tests.

Reply:
these are all quick, so this is ok

Reply:
Initially I only had 'uint8' and 'float64', but @jreback reasonably suggested to add some more. What would be a good balance here? If I'll remove usage of this fixture from all the unrelated tests like prefix / separator tests, and move None to separate stand-alone test, would ['uint8', 'i8', np.float64, bool] be OK? Still x4 number of tests, but each item uses a different way to specify dtype, so i think it's meaningful set of fixtures.

Reply:
i think toms point is to not apply this fixture to every test (just relevant ones)